src/v/datalake/record_schema_resolver.cc

```diff
@@ -96,7 +96,7 @@ struct schema_translating_visitor {
             auto type = type_to_iceberg(*d).value();
             co_return type_and_buf{
               .type = resolved_type{
-                .schema = *d,
```

Comment:
Can we also document this in the descriptor method? Thanks!

Reply:
Will do